index.js

```diff
@@ -2,17 +2,37 @@
 //.
 //. [Fluture][] type definitions for [Sanctuary][].
 //.
+//. ## Usage
+//.
 //. ```console
-//. $ npm install --save fluture sanctuary-def fluture-sanctuary-types
+//. $ npm install --save fluture-sanctuary-types
 //. ```
 //.
-//. ## Usage
+//. Note that you also need [Fluture][] and [sanctuary-def][] installed.
+//. Sanctuary-def comes preinstalled with Sanctuary, so you could install
+//. either one. Fluture has to be installed separately. See `package.json`
+//. for compatible versions (defined in `peerDependencies`).
+//.
+//. ### EcmaScript Module
 //.
 //. ```js
 //. import $ from 'sanctuary-def';
-//. import {env} from 'fluture-sanctuary-types';
+//. import {env, FutureType} from 'fluture-sanctuary-types/index.js';
+//. import {resolve} from 'fluture/index.js';
+//.
+//. $.create ({checkTypes: true, env: $.env.concat (env)});
```

Comment:
Without the `const def =` prefix, readers could be led to believe that `$.create` performs side effects.

Reply:
I updated the usage examples to use the sanctuary module instead of sanctuary-def, as I think it's the more common use-case. It also gave me an excuse to assign the output from `create` to a variable. See https://github.com/fluture-js/fluture-sanctuary-types/compare/7dd0f29773d3aefb460e74e3427db6d2f5008855..a4a6ca891e20d8da1712da670824ce2aa18bfa13 :)